classes/openrc.bbclass

```diff
@@ -41,23 +37,25 @@ openrc_postinst() {
 }
 
 openrc_prerm() {
-    if ! ${@bb.utils.contains('DISTRO_FEATURES', 'openrc', 'true', 'false', d)}; then
-        exit 0
-    fi
-
```

Comment:
Split into 3 commits please:
- Remove redudant DISTRO_FEATURES checks
- adding --ifexists (I'm guessing this is for opkg again? But isn't the script still there during prerm?)
- adding vardeps (though I'm not sure why we need this and systemd.bbclass doesn't?  Is this something that should be in the next commit?)

The questions in parens above you can just cover in the commit messages.

Reply:
Ok no problem.
- `--ifexists` is clearly not mandatory but is "free" and avoid an error if the service was running but a deleted script. I can also remove it if you prefer.
- if not specified in vardeps, any modification to these values will not trigger a new call to openrc_populate_packages then not rebuild the package with fresh configuration/data. I already checked systemd.bbclass and was surprised. I can only explain this by the fact it's really required when we are working on openrc support for a new recipe (because we modify multiple times the variables) but that's the key concept of vardeps.